test/unit/selendroid-specs.js

```diff
@@ -47,38 +48,38 @@ describe('SelendroidServer', () => {
 
   describe('#prepareModifiedServer', withMocks({adb, fs}, (mocks, S) => {
     let selendroid = new SelendroidServer(buildSelendroidOpts(adb));
-    it('should build a modified server if one doesnt exist', async () => {
+    it('should build a modified server if one doesnt exist', async function () {
       mocks.fs.expects("exists").once()
         .withExactArgs(selendroid.modServerPath)
-        .returns(Promise.resolve(false));
+        .returns(B.resolve(false));
```

Comment:
Is B.resolve call really necessary? `await` does not care much if the result is a promise or a simple data type

Reply:
It does not matter. Whoever wrote this originally did this, and it does not hurt.